src/styles/colors.css

```diff
@@ -13,30 +13,39 @@
     --white-six: #e7e7e7;
```

Comment:
Large diff, sorry, but it should just be the removal of the vars listed in the write up, renaming of the light-theme button vars and some re-organization.